Security/Authentication/Token/OAuth2Token.php

```diff
@@ -38,8 +39,8 @@ private function buildRolesFromScopes(): array
     {
         $roles = [];
 
-        foreach ($this->getAttribute('server_request')->getAttribute('oauth_scopes', []) as $scope) {
```

Comment:
@rjwebdev This seems like an accidental change. The attribute on the server request is really named `oauth_scopes`.

Reply:
Sorry, it was indeed by accident. Changed it back